src/main/java/nextstep/subway/line/domain/LineRepository.java

```diff
@@ -1,6 +1,12 @@
 package nextstep.subway.line.domain;
 
+import java.util.List;
+import org.springframework.data.domain.Pageable;
 import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
 
 public interface LineRepository extends JpaRepository<Line, Long> {
+
+    @Query(value = "select l from Line l where l.id >= ?1 ")
+    List<Line> findAll(Long id, Pageable pageable);
```

Comment:
아실 수도 있지만 다음과 같이 사용할 수도 있습니다 ☺️
```suggestion
    @Query(value = "select l from Line l where l.id >= :id ")
    List<Line> findAll(@Param("id") Long id, Pageable pageable);
```